example/Makefile

```diff
@@ -21,6 +21,7 @@ up: .terraform env.auto.tfvars
 
 down: .terraform env.auto.tfvars
 	tofu destroy -auto-approve
+	bash files/cleanup.sh
```

Comment:
After the destroy the cleanup.sh is also removed.

What do you think about using a destroy time provisioner in Terraform to run this instead? https://developer.hashicorp.com/terraform/language/resources/provisioners/syntax#destroy-time-provisioners

Reply:
What happens if we don't clean up the job ?

Reply:
Nevermind my original comment in this thread. I thought we were still starting the processes in terraform and not in make. As we use make for starting, we should also use it for stopping.

---

I am fine with this lingering around until the reboot, same behaviour on linux/systemd. We can also clean it up for both.

Reply:
I also added a cleanup step for `systemd` and refactored it into a single file. We needed the cleanup anyway before starting a new port-forward.

[66cc606](https://github.com/hetznercloud/kubernetes-dev-env/pull/19/commits/66cc606ec598db05716e1cffb9fccfad93f17711)